go.sum

```diff
@@ -129,6 +129,8 @@ gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
 gopkg.in/inf.v0 v0.9.1 h1:73M5CoZyi3ZLMOyDlQh031Cx6N9NDJ2Vvfl76EDAgDc=
 gopkg.in/inf.v0 v0.9.1/go.mod h1:cWUDdTG/fYaXco+Dcufb5Vnc6Gp2YChqWtbxRZE0mXw=
+gopkg.in/square/go-jose.v2 v2.5.0 h1:OZ4sdq+Y+SHfYB7vfthi1Ei8b0vkP8ZPQgUfUwdUSqo=
```

Comment:
Note to reviewers: this library is distributed under the Apache License 2.

[Later] I actually don't see this package used anywhere in this patch?

Reply:
Good catch, that was used in some POC code I threw away

Reply:
Its in some example code
```
$ go mod why gopkg.in/square/go-jose.v2
# gopkg.in/square/go-jose.v2
github.com/aws/amazon-eks-pod-identity-webhook/hack/self-hosted
gopkg.in/square/go-jose.v2

```